pkg/objectstate/rte/rte.go

```diff
@@ -130,13 +130,13 @@ type GeneratedDesiredManifest struct {
 	DaemonSet *appsv1.DaemonSet
 }
 
-type GenerateDesiredManifestUpdater func(mcpName string, gdm *GeneratedDesiredManifest) error
+type GenerateDesiredManifestUpdater func(mcpName string, gdm *GeneratedDesiredManifest, instance *nropv1.NUMAResourcesOperator) error
 
 func SkipManifestUpdate(gdm *GeneratedDesiredManifest) error {
 	return nil
 }
 
-func (em *ExistingManifests) State(mf rtemanifests.Manifests, updater GenerateDesiredManifestUpdater) []objectstate.ObjectState {
+func (em *ExistingManifests) State(mf rtemanifests.Manifests, instance *nropv1.NUMAResourcesOperator, updater GenerateDesiredManifestUpdater) []objectstate.ObjectState {
```

Comment:
`FromClient` already gets a `instance` parameter, I think we can safely compute the flag there and carry it in the `ExistingManifests` struct

Reply:
Done.